tests/manage/monitoring/prometheus/test_capacity.py

```diff
@@ -12,55 +12,88 @@
 log = logging.getLogger(__name__)
 
 
-@pytest.mark.parametrize(
-    argnames='interface',
-    argvalues=[
-        pytest.param(
-            'rbd',
-            marks=[
-                pytest.mark.polarion_id("OCS-899"),
-                pytest.mark.bugzilla('1809248'),
-                tier2,
-                gather_metrics_on_fail(
-                    'ceph_cluster_total_used_bytes', 'cluster:memory_usage_bytes:sum'
-                )
-            ]
+@pytest.mark.polarion_id("OCS-899")
+@pytest.mark.bugzilla('1809248')
+@tier2
+@gather_metrics_on_fail(
+    'ceph_cluster_total_used_bytes', 'cluster:memory_usage_bytes:sum'
+)
+def test_rbd_capacity_workload_alerts(workload_storageutilization_95p_rbd):
+    """
+    Test that there are appropriate alerts when ceph cluster is utilized
+    via RBD interface.
+    """
+    api = prometheus.PrometheusAPI()
+    measure_end_time = workload_storageutilization_95p_rbd.get('stop')
+
+    # Check utilization on 95%
+    alerts = workload_storageutilization_95p_rbd.get('prometheus_alerts')
+
+    if config.ENV_DATA.get('ocs_version') == '4.2':
+        nearfull_message = (
+            'Storage cluster is nearing full. Expansion is required.'
+        )
+        criticallfull_mesage = (
+            'Storage cluster is critically full and needs immediate expansion'
+        )
+    else:
+        # since OCS 4.3
```

Comment:
can be 4.4 or in future 4.5, so better to use 'elif == 4.3'

Reply:
I would change it only if the message wasn't valid for 4.3, 4.4 and 4.5. This way it works as expected.